README.md

```diff
@@ -59,6 +59,14 @@ in order of preference:
 
 More on how releases are conducted can be found [here](./RELEASES.md).
 
+### Support Policy
```

Comment:
Using ‘## Support Policy’ you make it look better, aligned with other header sizes

Reply:
438850209